src/main/java/com/minecolonies/util/Utils.java

```diff
@@ -376,6 +380,7 @@ public static int getMiningLevel(@Nullable ItemStack stack, @Nullable String too
         {
             return -1;
         }
+        //todo: use 'better' version of this thing
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)